### ISSUE TITLE
[Bug] Formatter breaks `repeat until` pattern

### ISSUE THREAD

**Issue by scambier** [japhib/pico8-ls#43]
To reproduce:

```lua
repeat
  local foo = "bar"
until true
```

Format the code, it's now transformed into

```lua
repeat
  local foo = "bar"until true
```

And it's more buggy with a comment:

```lua
repeat
  local foo = "bar"
  -- comment
until true
```

formats into

```lua
repeat
  local foo = "bar"until
  -- comment
  true
```

**Comment by japhib** [comment]
Thanks for the example! Should be a relatively straightforward fix

**Comment by japhib** [comment]
Fixed with 0.5.1
